src/Core/tests/Unit/ConfigurationTest.php

```diff
@@ -93,4 +93,17 @@ public function testRegionFromIniFiles()
         self::assertFalse($config->isDefault('region'));
         self::assertEquals('eu-central-1', $config->get('region'));
     }
+
+    public function testEndpointFromEnv()
+    {
```

Comment:
couldn't you use the existing provider at line 78 instead?

Reply:
Removed dedicated test and replaced with additional case in the provider. This case tests the same functionality as `fallback env` but for the newly added env. Do you want me to replace the env inside `fallback env`  with `AWS_ENDPOINT_URL` or is it okay to have both?